scripts/list_arduinos.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import sys
+
+from pocs.sensors import arduino_io
+from pocs.utils import rs232
+
+
+# Support testing by just listing the available devices.
+if __name__ == '__main__':
+    port_infos = rs232.get_serial_port_info()
+    if port_infos:
+        fmt = '{:20s} {:30s} {}'
+        print(fmt.format('Device', 'Manufacturer', 'Description'))
+        for pi in rs232.get_serial_port_info():
+            print(fmt.format(pi.device, pi.manufacturer, pi.description))
+        print()
+    devices = arduino_io.list_arduino_ports()
+    if devices:
+        print("Arduino devices: {}".format(", ".join(devices)))
+    else:
+        print("No Arduino devices found.")
+        sys.exit(1)
```

Comment:
I figured this could be used by another script that is doing setup of a unit, so distinguishing between found some devices and found no devices is worthwhile. But, not used yet.